app/code/Magento/Catalog/Block/Product/ListProduct.php

```diff
@@ -7,13 +7,17 @@
 namespace Magento\Catalog\Block\Product;
 
 use Magento\Catalog\Api\CategoryRepositoryInterface;
+use Magento\Catalog\Block\Product\Context;
 use Magento\Catalog\Block\Product\ProductList\Toolbar;
 use Magento\Catalog\Model\Category;
```

Comment:
Import of Magento\Catalog\Model\Category is never used.

Reply:
This is used on  prepareSortableFieldsByCategory method for $category parameter.